qutebrowser/completion/completer.py

```diff
@@ -151,6 +151,19 @@ def _get_completion_model(self, completion, parts, cursor_part):
             except KeyError:
                 # No completion model for this section/option.
                 model = None
+        elif completion == usertypes.Completion.keybinding:
+            # look for --mode to provide mode-specific binding completions
+            # since flags are ignored by most completers, we need to re-parse
+            parts, cursor_part = self._filter_cmdline_parts(self.split(),
+                                                            cursor_part,
+                                                            ignore_flags=False)
+            if parts[cursor_part - 1].startswith("--mode="):
```

Comment:
This doesn't account for `:bind --mode caret ...` (i.e. `--mode caret` instead of `--mode=caret`; both work). Do you see an easy way to make that work?